docs/wave.rst

```diff
@@ -0,0 +1,106 @@
+Acoustic Wave Modelling using Devito
+==================================================
+
+Consider the acoustic wave equation given in 3D:
+
+.. math::
+   m(x,y,z)\frac{\partial^2 u}{\partial t^2} + \eta(x,y,z)\frac{\partial u}{\partial t}-\frac{\partial^2 u}{\partial x^2}-\frac{\partial^2 u}{\partial y^2}-\frac{\partial^2 u}{\partial z^2}= 0
+
+   u(x,y,z,0) = 0
+
+   \frac{\partial u(x,y,z,t)}{\partial t}|_{t=0} = 0
+
+where :math:`\eta` represents damping, :math:`m` represents square slowness
+
+In this tutorial, the origin, spacing and true velocity are stored in instance of :obj:`IGrid()` called :obj:`model`.
+The gap between two consecutive timestep :obj:`dt` is :samp:`model.get_critical_dt()`.
+All the seismic data and initial settings like the coordinate of source,
+receiver coordinates are stored in an instance of :obj:`IShot()` called :obj:`data`::
+
```

Comment:
My intention is to explain the usage of IShot() and IGrid(). I thought IShot() is used to store(define?) the seismic data like source(q) and the receiver coordinates? Is it correct?